### PR TITLE
ROCm 6.3.0 - cherrypick

### PR DESCRIPTION
**Pull request by kiritigowda**
* Documentation Updates
* GFX 940/1 removal

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,101 +2,59 @@
...
-[https://rocm.docs.amd.com/projects/MIVisionX/en/latest/doxygen/html/index.html](https://rocm.docs.amd.com/projects/MIVisionX/en/latest/doxygen/html/index.html)
+The full documentation for MIVisionX is available at [https://rocm.docs.amd.com/projects/MIVisionX/en/latest/doxygen/html/index.html](https://rocm.docs.amd.com/projects/MIVisionX/en/latest/doxygen/html/index.html)
 
-## MIVisionX 3.1.0 (unreleased)
+##  MIVisionX 3.1.0 for ROCm 6.3.0
+
+### Changed
 
-### Changes
 * Setup: rocdecode install disabled
```
Comment (spolifroni-amd) [inline CHANGELOG.md:11]:
```suggestion
* Setup: rocDecode installation has been disabled.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,101 +2,59 @@
...
 * Package: rocdecode dependency removed
```
Comment (spolifroni-amd) [inline CHANGELOG.md:12]:
```suggestion
* The dependency on rocDecode has been removed.
```

**Review by spolifroni-amd — APPROVED** [review]
I've left two comments that are minor. In the case of removing rocDecode dependency from package, I took a guess at what it meant. If it isn't what was meant, just capitalize the D in rocDecode and we're good.

**Comment by LakshmiKumar23** [comment]
@kiritigowda will open a new PR

**Pull request closed without merging**